source/includes/stackpath/_workloads.md

```diff
@@ -160,6 +173,32 @@ Attributes | &nbsp;
 `type`<br/>*string* | Specify whether a workload is a VM-based workload or container-based.
 `userData`<br/>*string* | User data compatible with cloud-init to be run on the workload's instances.
+`livenessProbe`<br/>*Object* | A health check probe against a workload instance to determine if it is alive.
+`livenessProbe.initialDelaySeconds`<br/>*integer* | The initial delay before the probe starts.
+`livenessProbe.timeoutSeconds`<br/>*integer* | The number of seconds before the probe times out and is considered a failure.
+`livenessProbe.periodSeconds`<br/>*integer* | The frequency of the probe in seconds.
+`livenessProbe.successThreshold`<br/>*integer* | The minimum consecutive successes required before a probe is considered successful after a failure. This must be 1 for liveness probes.
+`livenessProbe.failureThreshold`<br/>*integer* | The amount of failures seen before the probe is considered a failure.
+`livenessProbe.httpGet`<br/>*Object* | HTTP GET request executed against an endpoint running on an instance.
+`livenessProbe.httpGet.path`<br/>*string* | The path portion of the URL to request.
+`livenessProbe.httpGet.port`<br/>*integer* | The TCP port to query in the HTTP request.
+`livenessProbe.httpGet.httpHeaders`<br/>*List* | A list of HTTPs headers.
+`livenessProbe.httpGet.scheme`<br/>*enum* | HTTP scheme to use in the HTTP request. It could be either "HTTP" or "HTTPS".
+`livenessProbe.tcpSocket`<br/>*Object* | The instance's TCP port to connect to.
+`livenessProbe.tcpSocket.port`<br/>*Object* | The TCP port to query.
+`readinessProbe`<br/>*Object* | A health check probe against a workload instance to determine if it is ready to receive traffic. A liveness is required for this to be set.
+`readinessProbe.initialDelaySeconds`<br/>*integer* | The initial delay before the probe starts.
+`readinessProbe.timeoutSeconds`<br/>*integer* | The number of seconds before the probe times out and is considered a failure.
+`readinessProbe.periodSeconds`<br/>*integer* | The frequency of the probe in seconds.
+`readinessProbe.successThreshold`<br/>*integer* | The minimum consecutive successes required before a probe is considered successful after a failure. This must be 1 for liveness probes.
+`readinessProbe.failureThreshold`<br/>*integer* | The amount of failures seen before the probe is considered a failure.
+`readinessProbe.httpGet`<br/>*Object* | HTTP GET request executed against an endpoint running on an instance.
+`readinessProbe.httpGet.path`<br/>*string* | The path portion of the URL to request.
+`readinessProbe.httpGet.port`<br/>*integer* | The TCP port to query in the HTTP request.
+`readinessProbe.httpGet.httpHeaders`<br/>*List* | A list of HTTPs headers.
+`readinessProbe.httpGet.scheme`<br/>*enum* | HTTP scheme to use in the HTTP request. It could be either "HTTP" or "HTTPS".
+`readinessProbe.tcpSocket`<br/>*Object* | The instance's TCP port to connect to.
+`readinessProbe.tcpSocket.port`<br/>*Object* | The TCP port to query.
```

Comment:
Should there be a mention that it is either httpGet or tcpSocket?

Reply:
at least for the create/update

Reply:
i'll add it

Reply:
d96ae48b3812b4333cdf73f2d55eeb98916e277d